src/line.rs

```diff
@@ -111,6 +106,15 @@ impl Lines {
         last.len = sym_len - (last.rva - sym_rva);
     }
 
+    pub(crate) fn pdb_finalize(&mut self, sym_rva: u32, sym_len: u32, address_map: &AddressMap) {
+        self.compute_len(sym_rva, sym_len);
+        self.compute_rva(address_map);
+    }
+
+    pub(crate) fn elf_finalize(&mut self, sym_rva: u32, sym_len: u32) {
```

Comment:
Does it make sense for `pdb_finalize` and `elf_finalize` to instead be implemented in a `finalize` method in the `Dumpable` trait?

Reply:
Agreed, that would keep all the platform-specific bits in their appropriate sources.